systemjs/index.d.ts

```diff
@@ -49,7 +53,7 @@ declare namespace SystemJSLoader {
          * Dependencies to load before this module. Goes through regular paths and map normalization.
          * Only supported for the cjs, amd and global formats.
          */
-        deps?: Array<string>;
+        deps?: string[];
```

Comment:
The purpose is simply to match the uniform style required by the tslint settings defined for DefinitelyTyped. Since PRs are now expected to run the linter I ran it and made the changes indicated. 
For example, arrays of more articulated types are required by the linter to use the full generic type syntax `Array<SomeInterface>`.
Personally, I always use the `[]` except where complying with existing guidelines.